src/app/view/components/_common/dot-page-selector/service/dot-page-selector.service.ts

```diff
@@ -0,0 +1,158 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { CoreWebService } from 'dotcms-js/dotcms-js';
+import { of } from 'rxjs/observable/of';
+
+export interface DotPageAsset {
+    template: string;
+    owner: string;
+    identifier: string;
+    friendlyname: string;
+    modDate: string;
+    cachettl: string;
+    pagemetadata: string;
+    languageId: number;
+    title: string;
+    showOnMenu: string;
+    inode: string;
+    seodescription: string;
+    folder: string;
+    __DOTNAME__: string;
+    sortOrder: number;
+    seokeywords: string;
+    modUser: string;
+    host: string;
+    lastReview: string;
+    stInode: string;
+    url: string;
+}
```

Comment:
why is not possible use 'DotPage' here?

Reply:
Too general. In fact we need to do something better with this, we need to create like `DotContentlet` and all this things should extend from that one.